### ISSUE TITLE
Misaligned characters in Japanese page

### ISSUE THREAD

**Issue by maekawatoshiki** [rust-lang/rust-www#927]
![2017-09-18 23 16 22](https://user-images.githubusercontent.com/9913176/30546594-012774d8-9cc8-11e7-9ef9-b28539c7e339.png)

- As you can see the image, **"開発に参加する"** is misaligned in Japanese page.

**Comment by steveklabnik** [comment]
https://github.com/rust-lang/rust-www/blob/master/_layouts/ja-JP/basic.html#L28-L33 is the source, looks like maybe a column tweak is needed.

**Comment by maekawatoshiki** [comment]
I see. But sorry, I have no confidence at tweaking HTML. I want someone to do it.

**Comment by steveklabnik** [comment]
Absolutely! I was trying to provide some help for anyone who wants to, that doesn't have to be you 😄

**Comment by magnonellie** [comment]
The only way I see this fixable is to either change the h2 margin or to change the text entirely. I'm probably wrong about that though. 0.8em seems to work pretty well. Thoughts?
